src/core/linalg/src/sparse/4C_linalg_multi_vector.cpp

```diff
@@ -97,32 +86,32 @@ void Core::LinAlg::MultiVector<T>::scale(double ScalarValue)
 template <typename T>
 void Core::LinAlg::MultiVector<T>::dot(const MultiVector& A, double* Result) const
 {
-  CHECK_EPETRA_CALL(vector_->Dot(A, Result));
+  CHECK_EPETRA_CALL(vector_->Dot(A.get_epetra_multi_vector(), Result));
 }
 
 template <typename T>
 void Core::LinAlg::MultiVector<T>::abs(const MultiVector& A)
 {
-  CHECK_EPETRA_CALL(vector_->Abs(A));
+  CHECK_EPETRA_CALL(vector_->Abs(*A.vector_));
```

Comment:
Why are we using the member variable here directly? Instead of e.g. `A.get_epetra_XXX()`.